pkgs/applications/video/mpv/wrapper.nix

```diff
@@ -79,6 +79,8 @@ let
         rm "$out/Applications/mpv.app/Contents/MacOS/mpv"
         makeWrapper "${mpv}/Applications/mpv.app/Contents/MacOS/mpv" "$out/Applications/mpv.app/Contents/MacOS/mpv" ${mostMakeWrapperArgs}
       '';
+
+      meta.mainProgram = "mpv";
```

Comment:
This should probably just inherit the other meta tags, too.

Reply:
Good idea! I've opened a PR: https://github.com/NixOS/nixpkgs/pull/147003